src/events/raw.ts

```diff
@@ -0,0 +1,13 @@
+import { bot, snakelize, VoiceServerUpdate, VoiceState } from "../../deps.ts";
+
+bot.eventHandlers.raw = function (data) {
+  if (data.t === "VOICE_SERVER_UPDATE") {
```

Comment:
Dont we have event handlers for these specifically now?

Reply:
no these are custom arguments passed for ux. But lavadeno requires the raw payload